python-stdlib/unittest/unittest.py

```diff
@@ -198,9 +204,9 @@ def run_class(c, test_result):
             try:
                 test_result.testsRun += 1
                 m()
-                print(" ok")
+                print(" PASS")
             except SkipTest as e:
-                print(" skipped:", e.args[0])
+                print(" SKIPPED:", e.args[0])
```

Comment:
These output format changes should be done in a separate commit - but only if the new format matches cpython unittest? I'd prefer to add this commit after the #488 changeset which has had some other changes made to the output format already to more closely match cpython.

Reply:
You are right, I reverted those text changes.